### ISSUE TITLE
Parameter support

### ISSUE THREAD

**Issue by jchristgit** [jb3/fizzbuzz.ko#1]
Please add support for configuring `max` on the command line when inserting the module, such that the buzzing can go on as long as desired.

**Comment by ChrisLovering** [comment]
i agree! This would be greatly beneficial to fizzbuzz.ko and its users.

if you are looking for implementation detail, this blog post might be helpful! https://tldp.org/LDP/lkmpg/2.4/html/x354.htm

**Comment by Xithrius** [comment]
I agree as well. Do it.

**Comment by jchristgit** [comment]
Hello @jb3, this lack of functionality is blocking implementation of this in my Fortune 500 company.

Could you please urgently respond to this matter?

Thank you
